test.ts

```diff
@@ -18,13 +15,13 @@ Deno.test('creates a presigned URL', () => {
   assertEquals(
     getSignedUrl(baseTestOptions),
     [
-      'https://examplebucket.s3.amazonaws.com/test.txt',
+      'https://s3.amazonaws.com/examplebucket/test.txt',
       '?X-Amz-Algorithm=AWS4-HMAC-SHA256',
       '&X-Amz-Credential=AKIAIOSFODNN7EXAMPLE%2F20130524%2Fus-east-1%2Fs3%2Faws4_request',
       '&X-Amz-Date=20130524T000000Z',
       '&X-Amz-Expires=86400',
       '&X-Amz-SignedHeaders=host',
-      '&X-Amz-Signature=aeeed9bbccd4d02ee5c0109b86d86835f995330da4c265957d157751f604d404',
+      '&X-Amz-Signature=733255ef022bec3f2a8701cd61d4b371f3f28c9f193a1f02279211d48d5193d7',
```

Comment:
The signature should match that provided by https://docs.aws.amazon.com/AmazonS3/latest/API/sigv4-query-string-auth.html#query-string-auth-v4-signing-example to easily verify correctness.

Reply:
I believe that this is related to the request style that is used.

Reply:
Yes, the official test cases should work.